pkg/config/setup/config_test.go

```diff
@@ -1100,3 +1099,72 @@ func TestProxyLoadedFromConfigFileAndEnvVars(t *testing.T) {
 	assert.Equal(t, proxyHTTPEnvVar, proxyHTTPConfig)
 	assert.Equal(t, proxyHTTPSEnvVar, proxyHTTPSConfig)
 }
+
+var testExampleConf = []byte(`
+secret_backend_command: some command
+additional_endpoints:
+  https://url1.com:
+    - ENC[api_key_1]
+    - ENC[api_key_2]
+  https://url2.eu:
+    - ENC[api_key_3]
+process_config:
+  additional_endpoints:
+    https://url1.com:
+      - ENC[api_key_1]
+      - ENC[api_key_2]
+    https://url2.eu:
+      - ENC[api_key_3]
+`)
```

Comment:
Could we remove the use of `ENC[]` ? When seeing the test case I first thought we were testing secrets replacement.

Reply:
Good idea, since this is only testing `configAssignAtPath`. Replaced the example config text.